src/txmempool.cpp

```diff
@@ -413,6 +413,7 @@ static CTxMemPool::Options&& Flatten(CTxMemPool::Options&& opts, bilingual_str&
 CTxMemPool::CTxMemPool(Options opts, bilingual_str& error)
     : m_opts{Flatten(std::move(opts), error)}
 {
+    m_txgraph = MakeTxGraph(64, 101'000, 10'000);
```

Comment:
nit 91d9bfcca62bd6c168476f6af442bd7049ff872e

I was going to complain about this magic number, maybe squash 83c8753abf9c2af75cfbacd5488605332da588a4 into its parent?

Reply:
This was squashed.